### PR TITLE
[dvsim] Some cosmetic updates 

### PR DESCRIPTION
**Pull request by sriyerg**
The first commit is an update that I forgot to add in PR #4429. 
The second prints the scratch path at the end again to make it easy for the user to get to it. 
The third is a cosmetic update to how the revision is retrieved and printed.

### CODE REVIEW

File: util/dvsim/dvsim.py [hunk 1]
```diff
@@ -383,10 +383,10 @@ def parse_args():
 
     rung.add_argument("--verbosity",
                       "-v",
-                      choices=['n', 'l', 'm', 'h', 'd'],
+                      choices=['n', 'l', 'm', 'h', 'f', 'd'],
                       metavar='V',
                       help=('Set tool/simulation verbosity to none (n), low '
-                            '(l), medium (m), high (h) or debug (d). '
+                            '(l), medium (m), high (h) full (f) or debug (d). '
```
Comment (rswarbrick) [inline util/dvsim/dvsim.py:389]: Nit: If re-pushing, this needs a comma before "full"

**Review by rswarbrick — APPROVED** [review]
Looks good. I wonder whether the branch printing in the second commit should actually be moved to the `FlowCfg` base class, but maybe that sort of refactor is for another day.

**Review by cindychip — APPROVED** [review]
LGTM

**Review by weicaiyang — APPROVED** [review]
LGTM

File: util/dvsim/dvsim.py (hunk 1 above)
Comment (sriyerg) [reply util/dvsim/dvsim.py]: Done.

**Pull request merged**